test/e2e/cluster_size_autoscaling.go

```diff
@@ -54,28 +60,58 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Feature:ClusterSizeAut
 		ReserveMemory(f, "memory-reservation", 1, memCapacityMb, false)
 		// Verify, that cluster size is not changed.
 		// TODO: find a better way of verification that the cluster size will remain unchanged.
-		time.Sleep(scaleTimeout)
-		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount, scaleTimeout))
-
+		time.Sleep(scaleUpTimeout)
+		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount, scaleUpTimeout))
 		framework.ExpectNoError(framework.DeleteRC(f.Client, f.Namespace.Name, "memory-reservation"))
-		framework.ExpectNoError(framework.WaitForClusterSize(f.Client, nodeCount, scaleTimeout))
+		framework.ExpectNoError(ResizeGroup(int32(nodeCount)))
```

Comment:
If we are testing scale down, we shouldn't be calling resize group. Scale down should resize automatically. Am I missing something?